manifests/s/Seonglae/Screencast/1.0.2/Seonglae.Screencast.yaml

```
@@ -0,0 +1,17 @@
+PackageIdentifier: Seonglae.Screencast
+Publisher: Seonglae Cho
+PackageName: Screencast
+PackageVersion: 1.0.2
+License: MIT
+InstallerType: exe
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
```

This is not needed as the InstallerType is Nullsoft
```suggestion
InstallerType: nullsoft
```